src/plugin.js

```diff
@@ -190,6 +196,7 @@ const onPlayerReady = (player, options) => {
         <h2 class="vjs-errors-headline">${this.localize(error.headline)}</h2>
           <div><b>${this.localize('Error Code')}</b>: ${(error.type || error.code)}</div>
           ${details}
+          <i><span id="fmsg" style="float:right;font-size:75%">${flashMessage}</span></i>
```

Comment:
We shouldn't be using `id` attributes anywhere because we need to consider cases of multiple players on a page. Also, I think the CSS should be in the `plugin.scss` file.